src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+.card {
+  width: 200px;
+  height: 406px;
```

Comment:
- Don't fix container size (if there is no such a requirement) - in this case don't hardcode height
<img width="578" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/63243136/d004d9d5-a8db-42a2-8a29-2ec61cb49a34">